lib/puppet/provider/firewalld.rb

```diff
@@ -119,10 +119,7 @@ def available?
 
   def self.available?
     check_running_state if state.nil?
-    if state.nil?
-      return false
-    else
-      return true
-    end
+    return false if state.nil?
```

Comment:
did you consider a ternary? I'm not sure if rubocop likes that as well:
```suggestion
    state.nil? ? false : true
```

(untested)

Reply:
or just `!state.nil?` ?

Reply:
that sounds even better

Reply:
Updated.